uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -309,18 +350,25 @@ sub get_acquisitions {
 ################################################################
 sub computeMd5Hash {
     my $this = shift;
-    my ($file) = @_;
-    $this->{LOG}->print(
-        "==> computing md5 hash for MINC body.\n"
-    ) if $this->{verbose};
+    my ($file, $tarchive_id) = @_;
```

Comment:
in 2 or 3 instances like here in computeMd5Hash(), the function has been changed to pass on the $tarchive_id (so we can spool the message with the needed information). Please verify this is Okay.